unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -16,12 +16,12 @@ class CreatorLockForm extends React.Component {
   constructor (props, context) {
     super(props)
     this.state = {
-      keyReleaseMechanism: 0, // Public
-      expirationDuration: 30,
-      expirationDurationUnit: 86400, // Days
+      keyReleaseMechanism: '0', // Public
```

Comment:
Nah, I don't think this is a problem and I actually think we need to keep the initial types here :(